README.md

```diff
@@ -10,3 +10,21 @@
 
 1. [Programming Languages](Professional_Skills_0002/Programming_Languages_%232_0002.md)
 2. [Web Development](Professional_Skills_0002/Web_Development_%232_0002.md)
+
+## Naphtal 0003
+
+### General Skills
+
+1. [Communication Skills](General_Skills_0003\Communication_%231_0003.md)
+
+2. [Negotiation Skills](General_Skills_0003\Negotiation_Skills_%231_0003.md)
+
+3. [Social Skills](General_Skills_0003\Social_Skills_%231_0003.md)
+
+### Professional Skills
+
```

Comment:
Please, review and test your links (All Links in this section) they are not redirecting to the intended page.